content/en/docs/reference/node/device-plugin-api-versions.md

```diff
@@ -0,0 +1,29 @@
+---
+content_type: "reference"
+title: Kubelet Device Manager API Versions
+weight: 10
+---
+
+Please note that the device plugin API is present here: https://github.com/kubernetes/kubelet/tree/master/pkg/apis/deviceplugin.
+
+#### Compatibility matrix
+
+|               |  Kubernetes 1.21 | Kubernetes 1.22 | Kubernetes 1.23 | Kubernetes 1.24 | Kubernetes 1.25 | Kubernetes 1.26 |
```

Comment:
I recommend flipping this table around: columns for alpha, beta, etc and rows for Kubernetes minor versions.

That will make it easier to maintain.

Reply:
Sure, updated.